javascript-timers/timeout.js

```diff
@@ -0,0 +1,8 @@
+var $greet = document.querySelector('.message');
+document.addEventListener('load', greeting);
```

Comment:
`greeting` should only run on the completion of the timeout, not when the `'load'` event fires on the document.